### PR TITLE
Add basic immunisation and booster vaccines to glossary

### LINKED ISSUE #3165: Add new terms reg. vaccination statistics to the Glossary
## Enhancement description
Please add the definitions of the following terms (and their German equivalents listed in brackets) to the [Glossary](https://www.coronawarn.app/en/faq/results/?search=&topic=glossary):
- "People Who Have Received at Least One Vaccine Dose" ("Mindestens eimal geimpfte Personen")
- "People Who Have Basic Immunization" ("Grundimmunisiert")
- "People Who Have Received a Booster Vaccination" ("Personen mit Auffrischimpfung")

## Problem and motivation
The app includes the above-mentioned terms without explaining them. It would be good to at least have a definition of the terms in the CWA website glossary. 

## Is this something you're interested in working on
No, this should be done by the team.

## Additional information
This is a follow-up to https://github.com/corona-warn-app/cwa-documentation/issues/922

---
Internal Tracking ID: [EXPOSUREAPP-14220](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14220)

### PR DESCRIPTION
**Pull request by brianebeling**
Fixes #3165

Texts are taken from: https://impfdashboard.de/en/

en
image

de
image
image

---
Internal Tracking ID: [EXPOSUREAPP-14220](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14220)

### CODE REVIEW

**Comment by GisoSchroederSAP** [comment]
Guys, this needs to be carefully reviewed to avoid we put something in the glossary, that is not in sync with the official definition of the legislative body. All input is welcome, but please don't merge before we have a "go" from the experts.
Thx in advance!

**Comment by GisoSchroederSAP** [comment]
The "Basic Immunization" (correct term: _Primary_ Immunization) needs to be adjusted please, because the current description does not cover the acceptable cases. Please use this explanation instead: 

> Als grundimmunisiert zählen alle Menschen, die die initiale Impfserie durchlaufen haben. Bei SARS-CoV-2 umfasst diese im Allgemeinen zwei Impfdosen, bei nachweislich Genesenen aber nur eine einzelne Impfdosis. Auch Personen, die bereits weitere Impfdosen erhalten haben, werden hier mitgezählt. Eine vollständige Grundimmunisierung ist nicht gleichbedeutend mit einem vollständigen Impfschutz.

image
(Source: Fachwörterbuch Infektionsschutz und Infektionsepidemiologie)

As a result, also the term "Auffrischimpfung" can be simplified:

> Als Geimpfte mit einer Auffrischimpfung zählen alle Menschen, die grundimmunisiert [hier ggf. Querverlinkung?] sind und danach mindestens eine weitere Impfdosis erhalten haben.

Thank you.

**Comment by MikeMcC399** [comment]
@GisoSchroederSAP 

If the term "Basic Immunization" is incorrect then it would need to be changed in the app. See https://www.coronawarn.app/en/screenshots/#android_stats.

**Comment by GisoSchroederSAP** [comment]
For consistency, let's stay with "basic communication", otherwise we generate new irritation and (in worst case) need to go through all screens with UI and translation again. ItÄs not worth this effort. I just wanted to mention the source where the definition for "Grundimmunisierung" comes from.

File: src/data/faq.json
Comment (MikeMcC399) [inline src/data/faq.json:3242]: Suggest to use the exact spelling used in the app i.e. Immunization, not Immunisation, also for the anchor. (The app normally uses US not UK spelling.)

File: src/data/faq_de.json
Comment (MikeMcC399) [inline src/data/faq_de.json:3455]: Suggest:
"anchor": "basic_immunisation" => "anchor": "basic_immunization"

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Please use US immunization spelling (with "z") as used in the app.
See for instance screenshots:

- https://www.coronawarn.app/en/screenshots/#android_stats and
- https://www.coronawarn.app/en/screenshots/#ios_stats

**Comment by MikeMcC399** [comment]
@GisoSchroederSAP 

> For consistency, let's stay with "basic communication", otherwise we generate new irritation and (in worst case) need to go through all screens with UI and translation again. ItÄs not worth this effort. I just wanted to mention the source where the definition for "Grundimmunisierung" comes from.

I guess you meant to write "basic immunization" not "basic communication"!

I found that the UK National Health Service (NHS) uses the term "primary immunization" in agreement with your dictionary, however I agree that it is probably too much effort to request and get approval to use the term "primary immunization" everywhere instead. 

"basic immunization" is not only used on the statistics tiles in the app it is also used in the booster notification rule [BNR-DE-0400](https://timokoenig.github.io/eu-dcc-rules/?ruleset=de-bn-rules&country=DE&rule=BNR-DE-0400) "The German Standing Committee on Vaccinations (STIKO) recommends a second booster vaccination for everyone age 60 and older who had their first booster vaccination more than 6 months ago. This recommendation is also valid for persons who were diagnosed with coronavirus after their **basic immunization**, if the infection occurred more than 6 months ago."

**Comment by GisoSchroederSAP** [comment]
You're right, it should have been "basic immunization" (instead of _communication_ ). Let's see if we can gradually introduce the "primary immunization" term by just using it in the communication as often as possible.

**Comment by brianebeling** [comment]
> In addition the Federal Ministry of Health uses "basic immunisation" (UK spelling!) on their English-language website https://impfdashboard.de/en/:
> 
> image

That is also what I oriented myself at. That said, I will implement the requested changes and report back. Thanks!

**Comment by brianebeling** [comment]
What do you think about linking to the new glossary items from everywhere?

Like this for example:
image

It improves navigation, but readability or aesthetics may suffer a bit from a lot of links.

File: src/data/faq.json
Comment (MikeMcC399) [reply src/data/faq.json:3250]: This conversation is marked as resolved although there are still discrepancies in the spelling between the header (immunisation) and the text (immunization).

File: src/data/faq.json
Comment (brianebeling) [reply src/data/faq.json:3250]: Sorry, they are resolved locally, but I didn't push yet. I will try to not resolve comments, unless pushed, in the future.

Comment (MikeMcC399) [reply src/data/faq.json:3250]: > I will try to not resolve comments, unless pushed, in the future.

That would be better, to keep the status and the PR contents in sync.

File: src/data/faq.json
Comment (MikeMcC399) [inline src/data/faq.json:3251]: "to be primed" sounds strange in this context. See https://dictionary.cambridge.org/dictionary/english/prime (verb)

suggest as replacement:
"to have completed basic immunization"

**Comment by MikeMcC399** [comment]
@brianebeling 

This PR is still in draft status, so not inviting reviews at this time, however I wanted to mention that there are multiple malformed links with `<a/>` which should be `</a>`.

Cypress will check these for you and fail, especially the `faq_link_attr` test.

**Comment by MikeMcC399** [comment]
@brianebeling 

> What do you think about linking to the new glossary items from everywhere?

Good idea. This needs to be done also for the English FAQ, not just the German one though.

**Comment by MikeMcC399** [comment]
@brianebeling 

For easier reading please mark the appropriate conversations above as [resolved](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews#resolving-conversations).

File: src/data/faq.json
Comment (MikeMcC399) [inline src/data/faq.json:518]: Put the "s" from "vaccinations" inside the link

File: src/data/faq.json
Comment (MikeMcC399) [inline src/data/faq.json:741]: It looks strange here as well.
Change to `vaccinations</a>`

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
In English I suggest not to split the link when a plural is involved:
i.e. make it `vaccinations</a>`

I marked up several places, but not all of them.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Same comment for `Auffrischimpfung</a>en` 
Put the "en" inside the link.

File: src/data/faq_de.json
Comment (MikeMcC399) [inline src/data/faq_de.json:518]: This is a link inside a link. I don't think that makes sense!

**Review by GisoSchroederSAP — APPROVED** [review]
Agreed, put the plural form for booster vaccination inside the link to the glossary, even if the glossary entry is the singular. Whoever has a problem to distinguish, I will answer, that the explanation for a single booster event does not differs at all for multiple booster vaccinations.

On the "basic immunization", IMHO there is no plural 's' required, 'cause, each person can only reach the basic immunization once. afterwards it will never be a "basic" but always be an "booster" immunization, right?

**Comment by MikeMcC399** [comment]
@GisoSchroederSAP 

> On the "basic immunization", IMHO there is no plural 's' required, 'cause, each person can only reach the basic immunization once. afterwards it will never be a "basic" but always be an "booster" immunization, right?

There is no occurrence of "immunizations" (in the plural) on the website, so this isn't an issue.

I am puzzled why you approved this PR. Normally approval is only given when the contents are 100% correct and ready to be merged. Was that the signal you intended to give?

**Comment by MohamedZaghdoudi** [comment]
hey @MikeMcC399  and  @GisoSchroederSAP,  thanks for the feedback.  
I have already corrected it and uploaded again .

**Review by larswmh — APPROVED** [review]

**Comment by MikeMcC399** [comment]
Please remove or replace the screen-shots in the original post as they are outdated.

File: src/data/faq.json
Comment (MikeMcC399) [inline src/data/faq.json:999]: This is an illegal nested link. It should be restored to the original without the nested glossary link.

File: src/data/faq.json
Comment (MikeMcC399) [inline src/data/faq.json:1009]: Illegal nested link!

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
There are still illegal nested links in this PR. They need to be removed.

**Comment by MikeMcC399** [comment]
Please mark resolved conversations as resolved, so it is clearer which items still need to be addressed.

File: src/data/faq_de.json
Comment (larswmh) [reply src/data/faq_de.json]: Fixed in https://github.com/corona-warn-app/cwa-website/pull/3179/commits/167908cf4a3c35b3fad2d6bd099d7f90418dc868, thanks!

File: src/data/faq.json
Comment (larswmh) [reply src/data/faq.json]: Fixed in https://github.com/corona-warn-app/cwa-website/pull/3179/commits/167908cf4a3c35b3fad2d6bd099d7f90418dc868, thanks!

File: src/data/faq.json
Comment (larswmh) [reply src/data/faq.json]: Fixed in https://github.com/corona-warn-app/cwa-website/pull/3179/commits/167908cf4a3c35b3fad2d6bd099d7f90418dc868, thanks!

**Comment by larswmh** [comment]
Thanks @MikeMcC399 for your extensive review. I believe my latest commit should've sorted out all remaining issue on this PR. Conversations have been closed as suggested.

File: src/data/faq.json
Comment (MikeMcC399) [reply src/data/faq.json]: This one is resolved.

File: src/data/faq_de.json
Comment (MikeMcC399) [inline src/data/faq_de.json:912]: Another illegal nested link here.

**Comment by MikeMcC399** [comment]
@larswmh 

It's very difficult to check the HTML by hand and spot all the issues. In the end I used https://validator.w3.org/ and looked for the error message:

`Start tag a seen but an element of the same type was already open.`

File: src/data/faq_de.json
Comment (larswmh) [reply src/data/faq_de.json]: Fixed in https://github.com/corona-warn-app/cwa-website/pull/3179/commits/8730e3fa06233ad08580e93253c6a7e9f2f6fb6a, thanks!

File: src/data/faq_de.json
Comment (larswmh) [reply src/data/faq_de.json]: Fixed in https://github.com/corona-warn-app/cwa-website/pull/3179/commits/8730e3fa06233ad08580e93253c6a7e9f2f6fb6a, thanks!

**Comment by larswmh** [comment]
@MikeMcC399 

That is a very useful tool, thanks for sharing. After checking with the changes made in the latest commit, I could not find any more issues regarding nested links. Can you confirm?

**Review by MikeMcC399 — APPROVED** [review]
@larswmh 

> After checking with the changes made in the latest commit, I could not find any more issues regarding nested links. Can you confirm?

- I can't see any more link issues from this PR, so I suggest to go ahead and merge.

- I did notice a link issue using the same tool, so I submitted a separate PR https://github.com/corona-warn-app/cwa-website/pull/3207 to fix that one

**Pull request merged**
